content/redirects.txt

```diff
@@ -599,3 +599,267 @@
 /docs/providers/type/monitor-index.html        /docs/providers/index.html
 /docs/providers/type/network-index.html        /docs/providers/index.html
 /docs/providers/type/vcs-index.html            /docs/providers/index.html
+
+# Removing some ghost pages
+/docs/backends/index.html           /docs/language/settings/backends/index.html
+/docs/backends/config.html          /docs/language/settings/backends/configuration.html
+/docs/backends/init.html            /docs/language/settings/backends/index.html
+/docs/backends/operations.html      /docs/language/settings/backends/index.html
+/docs/backends/types/index.html     /docs/language/settings/backends/index.html
+/docs/state/environments.html       /docs/language/state/workspaces.html
+/docs/providers/terraform/index.html    /docs/language/state/remote-state-data.html
```

Comment:
This seems like the better redirect?
```suggestion
/docs/providers/terraform/index.html    /docs/language/providers/index.html
```

Reply:
Nah, not so much -- this was the special `terraform` built-in provider, and the only reason anyone would ever end up on its front page was because they were trying to look up the `terraform_remote_state` data source. Basically that front page only served to waste your time en route to the data source page, so that's where I pointed it.